zinit-autoload.zsh

```diff
@@ -3337,6 +3339,7 @@ EOF
           }
       }
       noglob +zinit-message  {p}-- ./configure --{rst}
+      local -x CPPFLAGS CFLAGS LDFLAGS
```

Comment:
Any reason to not just prefix the line below with this?

Reply:
zsweep doesn't yet recognize in-place temporary export of variables when they're prefixing a command, so I've added the `local -x …` line to workaround this. IMO it's a graceful workaround.